dev-tools/gen-target-info/src/main.rs

```diff
@@ -1,41 +1,11 @@
-use gen_target_info::{get_target_specs_from_json, write_target_tuple_mapping, RustcTargetSpecs};
-use std::{collections::BTreeMap, fs::File, io::Write as _};
+use std::{fs::File, io::Write as _};
 
 const PRELUDE: &str = r#"//! This file is generated code. Please edit the generator
 //! in dev-tools/gen-target-info if you need to make changes.
 
 "#;
 
-fn generate_riscv_arch_mapping(f: &mut File, target_specs: &RustcTargetSpecs) {
-    let riscv_target_mapping = target_specs
-        .0
-        .iter()
-        .filter_map(|(target, target_spec)| {
-            let arch = target.split_once('-').unwrap().0;
-            (arch.contains("riscv") && arch != target_spec.arch)
-                .then_some((arch, &*target_spec.arch))
-        })
-        .collect::<BTreeMap<_, _>>();
-    write_target_tuple_mapping(f, "RISCV_ARCH_MAPPING", &riscv_target_mapping);
-}
-
-fn generate_windows_triple_mapping(f: &mut File, target_specs: &RustcTargetSpecs) {
-    let windows_target_mapping = target_specs
-        .0
-        .iter()
-        .filter_map(|(target, target_spec)| {
-            let rust_target_parts = target.splitn(4, '-').collect::<Vec<_>>();
-            let os = *rust_target_parts.get(2)?;
-            (os.contains("windows") && target != &*target_spec.llvm_target)
-                .then_some((&**target, &*target_spec.llvm_target))
-        })
-        .collect::<BTreeMap<_, _>>();
-    write_target_tuple_mapping(f, "WINDOWS_TRIPLE_MAPPING", &windows_target_mapping);
-}
-
 fn main() {
```

Comment:
The generator is useless now, though I'll wait with removing it until we decide if we should be generating stuff in https://github.com/rust-lang/cc-rs/pull/1225/files#r1781935441.